docker-compose.yml

```diff
@@ -8,11 +8,21 @@ services:
     restart: always
     environment:
       MYSQL_DATABASE: puresynth
-      MYSQL_USER: ${DB_USERNAME}
-      MYSQL_PASSWORD: ${DB_PASSWORD}
+      MYSQL_USER: puresynth
+      MYSQL_PASSWORD: puresynth
       MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
     ports:
       - "3307:3306"
 
+  flyway:
```

Comment:
We were supposed to do without this container. I added changes before to show you how to achieve that.